docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3.8"
+
+services:
+  node_obs_stream_widgets:
+    build: ./
+    image: obs_stream_widgets
+    container_name: obs_stream_widgets
+    command: [bash, -c, "yarn start"]
+    ports:
```

Comment:
Must expose 3000 port to allow the socket server to work

Reply:
Done